modules/heat_conduction/src/materials/GapConductance.C

```diff
@@ -124,6 +124,7 @@ GapConductance::GapConductance(const InputParameters & parameters)
     _gap_temp_value(_quadrature ? _zero : coupledValue("gap_temp")),
     _gap_conductance(declareProperty<Real>("gap_conductance" + _appended_property_name)),
     _gap_conductance_dT(declareProperty<Real>("gap_conductance" + _appended_property_name + "_dT")),
+    _gap_conductivity_property(declareProperty<Real>("gap_conductivity")),
```

Comment:
Would you mind either adding a new test or updating an existing test so we can keep this small change working?
Thanks